src/app/layout.tsx

```diff
@@ -86,7 +86,19 @@ export const metadata: Metadata = {
     title: "芝浦工業大学 デジクリ",
     siteName: "芝浦工業大学 デジクリ",
     description: "芝浦工業大学の公認サークル『デジクリ』の Web サイトです。",
-    images: "/ogp.png",
+    images: [
+      {
+        url: "/ogp.png",
+        width: 1200,
+        height: 630,
+        alt: "デジクリ",
```

Comment:
OGP 画像の `alt` 属性という観点から，ロゴの文字を書き起こすことが望ましい気がします。

Ref. <https://zenn.dev/harryduck/articles/alt-attribute-in-a11y#%E5%95%86%E5%93%81%E7%94%BB%E5%83%8F%E3%81%A8%E3%83%AD%E3%82%B4>

```suggestion
        alt: "デジクリ / Digital Creation Circle",
```